Compression/src/compressclient.java

```diff
@@ -1,3 +1,5 @@
+package Compression.src;
+
 
 public class compressclient {

```

Comment:
Can you change the name of the class according to the Java code conventions

Reply:
yep , done